pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[project]
+name = "troi"
+authors = [
+  { name = "MetaBrainz Foundation", email="support@metabrainz.org"}
+]
+description="An empathetic music recommendation system pipeline"
+readme = "README.md"
+requires-python=">=3.8"
+classifiers=[
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+  "Operating System :: OS Independent",
+]
+dynamic = ["version"]
+dependencies = [
+  'click>=8.0',
+  'ujson>=5.4.0',
+  'requests',
+  'psycopg2-binary>=2.9.3',
+  'countryinfo>=0.1.2',
+  'pylistenbrainz@git+https://github.com/metabrainz/pylistenbrainz.git@v0.5.4',
+  'python-dateutil>=2.8.2',
+  'spotipy>=2.22.1',
+  'more_itertools',
+  'pyparsing'
+]
+
+[project.scripts]
+troi = 'troi.li:cli'
+
+[project.optional-dependencies]
+tests = [
+  'pytest == 7.1.2',
+  'requests-mock'
```

Comment:
I'd always pin those dependencies, however I didn't wanted to change this without notice

Reply:
Fine with me, but why do you suggest this?

Reply:
@mayhem I just don't like messing around with tests that fail because a new version of a library breaks. Please note that not adding any version constraint means any currently found version might be installed.

The best scenario for me is pinning the dependencies and enabling the renovate bot in order to get a PR with a version bump, which then runs all tests so we can verify this works.